tests/wrapper_validator.py

```diff
@@ -27,3 +30,31 @@ def validate_wrapped_regression_model(wrapped_model, X_test):
     predictions = wrapped_model.predict(X_test)
     # validate predictions have correct shape
     assert len(predictions.shape) == 1
+
+
+def validate_wrapped_tf_model(wrapped_tf_model, X_test, model_task):
+    assert isinstance(wrapped_tf_model, WrappedTensorflowModel)
+    validate_wrapped_pred_classes_model(wrapped_tf_model, X_test, model_task)
+
+
+def validate_wrapped_pytorch_model(wrapped_pytorch_model, X_test, model_task):
+    assert isinstance(wrapped_pytorch_model, WrappedPytorchModel)
+    validate_wrapped_pred_classes_model(wrapped_pytorch_model, X_test, model_task)
+
+
+def validate_wrapped_pred_classes_model(wrapped_tf_model, X_test, model_task):
```

Comment:
nit: could update `wrapped_tf_model` to `wrapped_model` as it is not only for tensorflow model